src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -0,0 +1,2508 @@
+
+            private delegate Status GdipIsVisibleRegionPointI_delegate(IntPtr region, int x, int y, IntPtr graphics, out bool result);
+            private static FunctionWrapper<GdipIsVisibleRegionPointI_delegate> GdipIsVisibleRegionPointI_ptr;
+            internal static Status GdipIsVisibleRegionPointI(IntPtr region, int x, int y, IntPtr graphics, out bool result) => GdipIsVisibleRegionPointI_ptr.Delegate(region, x, y, graphics, out result);
+
+            private delegate Status GdipIsVisibleRegionRect_delegate(IntPtr region, float x, float y, float width, float height, IntPtr graphics, out bool result);
+            private static FunctionWrapper<GdipIsVisibleRegionRect_delegate> GdipIsVisibleRegionRect_ptr;
+            internal static Status GdipIsVisibleRegionRect(IntPtr region, float x, float y, float width, float height, IntPtr graphics, out bool result) => GdipIsVisibleRegionRect_ptr.Delegate(region, x, y, width, height, graphics, out result);
+
+            private delegate Status GdipIsVisibleRegionRectI_delegate(IntPtr region, int x, int y, int width, int height, IntPtr graphics, out bool result);
+            private static FunctionWrapper<GdipIsVisibleRegionRectI_delegate> GdipIsVisibleRegionRectI_ptr;
+            internal static Status GdipIsVisibleRegionRectI(IntPtr region, int x, int y, int width, int height, IntPtr graphics, out bool result) => GdipIsVisibleRegionRectI_ptr.Delegate(region, x, y, width, height, graphics, out result);
+
+            private delegate Status GdipCombineRegionRect_delegate(IntPtr region, ref RectangleF rect, CombineMode combineMode);
+            private static FunctionWrapper<GdipCombineRegionRect_delegate> GdipCombineRegionRect_ptr;
+            internal static Status GdipCombineRegionRect(IntPtr region, ref RectangleF rect, CombineMode combineMode) => GdipCombineRegionRect_ptr.Delegate(region, ref rect, combineMode);
+
+            private delegate Status GdipCombineRegionRectI_delegate(IntPtr region, ref Rectangle rect, CombineMode combineMode);
+            private static FunctionWrapper<GdipCombineRegionRectI_delegate> GdipCombineRegionRectI_ptr;
+            internal static Status GdipCombineRegionRectI(IntPtr region, ref Rectangle rect, CombineMode combineMode) => GdipCombineRegionRectI_ptr.Delegate(region, ref rect, combineMode);
+
+            private delegate Status GdipCombineRegionPath_delegate(IntPtr region, IntPtr path, CombineMode combineMode);
+            private static FunctionWrapper<GdipCombineRegionPath_delegate> GdipCombineRegionPath_ptr;
+            internal static Status GdipCombineRegionPath(IntPtr region, IntPtr path, CombineMode combineMode) => GdipCombineRegionPath_ptr.Delegate(region, path, combineMode);
+
+            private delegate Status GdipGetRegionBounds_delegate(IntPtr region, IntPtr graphics, ref RectangleF rect);
+            private static FunctionWrapper<GdipGetRegion
```

Comment:
Only the ones I touched manually are wrapped neatly... the rest were the raw output from a converter tool I wrote. I can see if I can format all of these automatically somehow.